src/frontends/onnx/frontend/src/utils/arg_min_max_factory.cpp

```diff
@@ -93,7 +99,13 @@ std::shared_ptr<ov::Node> ArgMinMaxFactory::make_topk_subgraph(v11::TopK::Mode m
         return result;
     }
 
-    const auto topk = std::make_shared<v11::TopK>(m_input_node, k_node, m_axis, mode, v11::TopK::SortType::NONE);
+    const auto topk = std::make_shared<v11::TopK>(m_input_node,
+                                                  k_node,
+                                                  m_axis,
+                                                  mode,
+                                                  v11::TopK::SortType::SORT_VALUES,
+                                                  element::i32,
+                                                  true);
 
     const auto result = std::make_shared<v0::Convert>(topk->output(1), ov::element::i64);
```

Comment:
You can also remove this convert if specify i64 as output type of TopK

Reply:
That is true, but same is true for topk with i64 case, it should be converted to i32 version, but we will create 1 less node, which is better I think